_posts/2015-06-10-agile-fluency.md

```diff
@@ -0,0 +1,24 @@
+---
+layout: post
+title: "When a company turns agile"
+date: "2015-06-10 07:28:00"
+icon: refresh
+tags: agile scrum company
+categories: scrum
+authors: ["Anja Bärike"]
+---
+
+Did you ever wonder how companies that are going the agile way are developing further over time? And how long it actually takes to get from level to level? Or how many companies achieve certain levels of agility?
+
+Here are all the answers:
+
+{% image blog-agile-fluency.jpg %}{% endimage %}
+
+The picture shows the so called Agile Fluency Model. The numbers are based on questions that had been asked to companies who changed to an agile methodology some years ago.
+The most impressive numbers are probably, that it takes really long to achieve two or three stars.
+It is astonishing that only 5% of all companies who decide to go the agile way achieve a three star model.
+
+ePages introduced the agile system 1.5 years ago and is right on track between the first and the second star.
```

Comment:
ePages introduced the agile system approx. 1.5 years ago and is right on track, between the first and the second star.